src/simplify_reshapes.cpp

```diff
@@ -806,6 +806,27 @@ struct find_transpose_slice
     }
 };
 
+// replace scalar (multibroadcast ->unsqueeze, etc.) with just an updated (multibroadcast)
+struct find_scalar_multibroadcast_modify
```

Comment:
I think it would be better to name it as `find_scalar_multibroadcast_reshape_or_transpose`.